tests/schema/tests.py

```diff
@@ -2123,6 +2123,7 @@ def get_field(*args, field_class=IntegerField, **kwargs):
                     "using": "",
                     "columns": editor.quote_name(column),
                     "extra": "",
+                    "condition": "",
```

Comment:
A test was failing without this change. I'm not sure why `Statement` was not used here.